### PR TITLE
Adding support for error handling

### PR DESCRIPTION
**Pull request by prebansa**

### CODE REVIEW

File: Tasks/ContainerBuildV0/src/utils.ts [hunk 1]
```diff
@@ -100,7 +100,10 @@ export async function getServiceDetails() {
     };
     var serviceResponse = kubectlTool.execSync(executionOption);
 
-    if (serviceResponse && serviceResponse.stdout) {
+    if (serviceResponse && serviceResponse.stderr) {
+        tl.setResult(tl.TaskResult.Failed, serviceResponse.stderr);
+    }
+    else if (serviceResponse && serviceResponse.stdout) {
         namespace = JSON.parse(serviceResponse.stdout).metadata.namespace ? JSON.parse(serviceResponse.stdout).metadata.namespace : namespace;
```
Comment (vithati) [inline Tasks/ContainerBuildV0/src/utils.ts]: Repeated multiple places. Assingn to variable and use in other places.

File: Tasks/ContainerBuildV0/src/utils.ts [hunk 2]
```diff
@@ -100,7 +100,10 @@ export async function getServiceDetails() {
     };
     var serviceResponse = kubectlTool.execSync(executionOption);
 
-    if (serviceResponse && serviceResponse.stdout) {
+    if (serviceResponse && serviceResponse.stderr) {
+        tl.setResult(tl.TaskResult.Failed, serviceResponse.stderr);
```
Comment (vithati) [inline Tasks/ContainerBuildV0/src/utils.ts]: throw exception with 'serviceResponse.stderr' main function will handle and set task failed.

File: Tasks/ContainerBuildV0/src/utils.ts (hunk 1 above)
Comment (prebansa) [reply Tasks/ContainerBuildV0/src/utils.ts]: made changes as suggested

File: Tasks/ContainerBuildV0/src/utils.ts (hunk 2 above)
Comment (prebansa) [reply Tasks/ContainerBuildV0/src/utils.ts]: Added throw exception

File: Tasks/ContainerBuildV0/task.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 0,
         "Minor": 160,
```
Comment (vithati) [inline Tasks/ContainerBuildV0/task.json]: 161

**Pull request merged**